src/main/java/mods/railcraft/common/blocks/single/TileEngine.java

```diff
@@ -410,4 +376,12 @@ public static EnergyStage fromOrdinal(int ordinal) {
         }
 
     }
+
+    @Optional.Interface(iface = "buildcraft.api.mj.IMjConnector", modid = "buildcraftlib")
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.

Reply:
I'm sorry, but your detection is wrong. What you've highlighted is an annotation, and following class name coventions, its name should be PascalCase, which already is. [Nm: Class names should start with an upper case letter (NM_CLASS_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-class-names-should-start-with-an-upper-case-letter-nm-class-naming-convention)